src/components/Icon/canonicalIconNames.tsx

```diff
@@ -125,6 +125,7 @@ export type ValidIconName =
     | "toggler-star-empty"
     | "toggler-star-filled"
     | "toggler-tree"
+    | "template-parameter"
```

Comment:
Do we await more `template-` icons? Otherwise this name sounds to "special", how could it be used in other interfaces or contextes? Could it be better to use something like:
* `artefact-template`
* `operation-usetemplate`
* `item-usetemplate` ?

Reply:
Maybe it would be better to just name it `template`. But if you prefer a hierarchical naming scheme, then maybe `form-template`?